library/src/main/res/anim/valid.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<set xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <scale
+        android:duration="250"
```

Comment:
What is the advantage of using the constant? I tried `shortAnimTime` but it was too fast and then `mediumAnimTime` was too slow. For this animation 250ms is the best duration time selected empirically ;)